lib/logger/src/logger.ts

```diff
@@ -0,0 +1,9 @@
+import winston from 'winston'
+import { format } from './format'
+import { consoleTransport } from './transports'
+
+export const rootLogger: winston.Logger = winston.createLogger({
+  level: process.env.LOG_LEVEL,
```

Comment:
I'd argue that  the defaults should be targeting the consumers of Tool Kit rather than the developers. They should have to think about manually setting the log level and hopefully there'll be a lot more of them than us in the future!

Reply:
oh, by 'developing with toolkit' I meant the consumers of toolkit! say if I'm running `npm run test` in next-toggler and have toolkit configured, I'd want to see that `npm run test` is running `jest --config=jest.config.ts` for example

Reply:
and perhaps we could add to the documentation that we could configure the env variable LOG_LEVEL for developers

Reply:
I think it's more a case of tweaking the log level of specific messages that you think are important for the consumer to see rather than changing the default log level

Reply:
fair enough, I think then we should not default to verbose but we log out everything that is run locally with 'info' but we can continue to log things that aren't run locally and run in the CI with 'verbose' as you are planning to set the CI to print verbose logs by default. 

eg. 
```logger.verbose(`Running: jest ${config.join(' ')}`)``` to info in jest,  and also ```logger.verbose('running lint-staged...')``` to info in lint-staged